mozilla_aws_cli/static/index.js

```diff
@@ -68,7 +70,27 @@ const shutdown = async () => {
 };
 
 const pollState = setInterval(async () => {
+    if (! state.backendInProgress) {
```

Comment:
I don't think we usually have a space after this operator?

Reply:
Done in 4c19b5d43a469e4c2fccef8538be1c2f00c7cb31